types/react-google-picker/index.d.ts

```diff
@@ -0,0 +1,30 @@
+// Type definitions for react-google-picker 0.1
+// Project: https://github.com/sdoomz/react-google-picker
+// Definitions by: Evan Broder <https://github.com/ebroder>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.0
```

Comment:
```suggestion
```
why this one was added? (minimum version)

Reply:
I think I was trying to figure out how to clear this warning:

```
WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.

You may find that it works just fine, or you may not.

SUPPORTED TYPESCRIPT VERSIONS: >=3.3.1 <4.8.0

YOUR TYPESCRIPT VERSION: 4.9.0-dev.20220811

Please only submit bug reports when using the officially supported version.
```

but it's not necessary - I'll drop it.

Reply:
4.8 is RC (next), and the current tooling in ecosystem, does not support it. DT is being used by TS team as proving ground for upcoming TS versions